governance/mixed.go

```diff
@@ -119,6 +129,62 @@ func (e *MixedEngine) assembleParams(headerParams *params.GovParamSet) *params.G
 	return p
 }
 
+func (e *MixedEngine) handleParamUpdate(old, new *params.GovParamSet) {
+	// TODO: key set must be the same, which is guaranteed at NewMixedEngine
```

Comment:
@ian0371 Is it TODO or NOTE?

Reply:
It was a TODO, but thanks to fallback mechanism now it's a NOTE. fixed in 196fbdc.